add-ons/hyper-sorter/index.js

```diff
@@ -20,8 +20,8 @@ function Hypersorter(grid, targets) {
 Hypersorter.prototype.name = 'Hypersorter';
 
 Hypersorter.prototype.install = function(targets) {
-    var Hypergrid = Object.getPrototypeOf(this.grid).constructor;
-    Hypergrid.properties.mixIn(require('./mix-ins/defaults'));
+    var Hypergrid = this.grid.constructor;
+    Hypergrid.defaults.mixIn(require('./mix-ins/defaults'));
```

Comment:
@Dwaynekj FYI, `Hypergrid.defaults` and `Hypergrid.properties` are synonyms. (Not the same as `grid.properties`!)